dice.js

```diff
@@ -0,0 +1,10 @@
+function rollDice(random1,random2, random3 ) {
+
+    random1 = Math.floor(Math.random()*6 )+1;
+    random2 = Math.floor(Math.random()*6 )+1;
+    random3 = Math.floor(Math.random()*6 )+1;
```

Comment:
if you're defining random1, random2, random3 within the function then you don't need to include it as a parameter
```js
function rollDice() {
//function body
}
```